adapters/src/main/kotlin/com/krzykrucz/elesson/currentlesson/lessonprogress/adapters/rest/LessonProgressDto.kt

```diff
@@ -0,0 +1,23 @@
+package com.krzykrucz.elesson.currentlesson.lessonprogress.adapters.rest
+
+import com.krzykrucz.elesson.currentlesson.lessonprogress.usecase.LessonProgress
+import com.krzykrucz.elesson.currentlesson.shared.LessonTopic
+
+data class LessonProgressDto(
+    val semester: Int,
+    val className: String,
+    val subject: String,
+    val date: String,
+    val topic: LessonTopic?,
```

Comment:
why `LessonTopic` is nullable?

Reply:
LessonTopic is set only if attendance is checked and I figured that LessonProgress could be loaded before that.